src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -25,6 +25,8 @@ type Props = {
 };
 
 const VersionsSidebar = ({ error, isLoading, parentName, versions, ...rest }: Props) => {
+    const MAX_VERSIONS = 1000;
+    const maxVersions = versions.length >= 1000;
```

Comment:
Should this follow the `showLimit` style convention of the other variables? Also, `>= MAX_VERSIONS`?